docs/4___common_content.adoc

```diff
@@ -460,3 +460,112 @@ This attribute gives the size of this dimension of the array as a fixed, unchang
 |sizeConnector a|
 This attribute references another connector by name, that gives the size of this dimension of the array connector, e.g. a structural parameter or a constant of the underlying component that gives the dimension size.
 |===
+
+[#ssc_metadata]
+==== MetaData Sequence
+
+image:images/SystemStructureCommon_GMetaData.png[image,width=597,height=196]
+
+The MetaData element allows the specification of additional meta data for a given model element.
+Multiple (or no) MetaData elements may be present in a given model element.
+
+[width="100%",cols="33%,67%",options="header",]
+|===
+|Element  |Description
+|MetaData |One or more instances of this element *CAN* be present to specify meta data related to the containing model element.
+See below for details.
+|===
+
+===== MetaData
+
+image:images/SystemStructureCommon_MetaData.png[image,width=369,height=543]
+
+This element specifies a source of meta data related to the containing model element.
+
+The MetaData element is structured by the following subordinated elements.
+
+[width="100%",cols="33%,67%",options="header",]
+|===
+|Element  |Description
+|Content  |An instance of this element *CAN* be present to contain inlined content of the model element meta data.
+If it is present, then the attribute `source` of the MetaData element *MUST NOT* be present.
+See below for details.
+|Signature|One or more instances of this element *CAN* be present to specify digital signature information on the meta data referenced by the enclosing MetaData element.
+It is left unspecified what types of signatures are used and/or available for now.
+See below for details.
+|===
+
+The following XML attributes are specified for the MetaData element:
+
+[width="100%",cols="28%,72%",options="header",]
+|===
+|Attribute |Description
+|kind a|
+This attribute indicates the kind of model element meta data that is referenced, i.e. what role it plays in relation to the model element being described.
+The kind can be `general` or `quality`, indicating general meta data or quality related meta data, respectively.
+|type a|
+This mandatory attribute specifies the MIME type of the meta data, which does not have a default value.
+If no specific MIME type can be indicated, then the type `application/octet-stream` is to be used.
+|source a|
+Optional attribute indicating the source of the meta data as a URI (cf. RFC 3986).
+For purposes of the resolution of relative URIs the base URI is the URI of the SSD, if the sourceBase attribute is not specified or is specified as `SSD`, and the URI of the containing model element (e.g. component) if the base attribute is specified as `component`.
```

Comment:
```suggestion
The sourceBase attribute is used for the resolution of relative URIs.
If the sourceBase attribute is not specified or is specified as `SSD`, the base URI is the URI of the SSD.
If the sourceBase attribute is specified as `component`, the base URI is the URI of the containing model element (e.g. component).
```

Reply:
2024-09-27: Should be clarified by removing most of the duplicated content from sourceBase. This is to be handled in a separate PR which addresses the same language occuring in all instances of source/sourceBase.